src/pyodide/BUILD.bazel

```
@@ -14,6 +15,12 @@ copy_file(
     out = "generated/python_stdlib.zip",
 )
 
+copy_file(
+    name = "pyodide_packages_file.zip@rule",
```

There's a few places where it says zip that should probably say tar
```suggestion
    name = "pyodide_packages_file.tar@rule",
```

---

for the sake of CI expediency, I'll fix this in another PR (where I will add fastapi packages too)